packages/prompts/src/index.test.ts

```diff
@@ -322,6 +322,42 @@ describe.each(['true', 'false'])('prompts (isCI = %s)', (isCI) => {
 				prompts.settings.messages.error = originalErrorMessage;
 			});
 		});
+
+		describe('progress', () => {
```

Comment:
this should be in its own describe block rather than inside the `spinner` one

its ok to repeat the `beforeEach`/`afterEach`